_includes/v22.1/ui/statement-details.md

```diff
@@ -0,0 +1,150 @@
+
+To display the plan that was executed, click a plan ID. When you click the plan ID `13929609339379026000`, the following plan displays:
+
+<img src="{{ 'images/v22.1/ui_statement_plan.png' | relative_url }}" alt="Plan table" style="border:1px solid #eee;max-width:80%" />
+{% endif %}
+
+### Diagnostics
+
+The **Diagnostics** tab allows you to activate and download diagnostics for a SQL statement fingerprint.
+
+{{site.data.alerts.callout_info}}
+The **Diagnostics** tab is not enabled for {{ site.data.products.serverless }} clusters.
+{{site.data.alerts.end}}
+
+When you activate diagnostics for a fingerprint, CockroachDB waits for the next SQL query that matches this fingerprint to be run on any node. On the next match, information about the SQL statement is written to a diagnostics bundle that you can download. This bundle consists of [statement traces]({{ link_prefix }}show-trace.html) in various formats (including a JSON file that can be [imported to Jaeger]({{ link_prefix }}query-behavior-troubleshooting.html#visualize-statement-traces-in-jaeger)), a physical query plan, execution statistics, and other information about the query. The bundle contents are identical to those produced by [`EXPLAIN ANALYZE (DEBUG)`]({{ link_prefix }}explain-analyze.html#debug-option). You can use the information collected in the bundle to diagnose problematic SQL statements, such as [slow queries]({{ link_prefix }}query-behavior-troubleshooting.html#query-is-always-slow). We recommend that you share the diagnostics bundle with our [support team]({{ link_prefix }}support-resources.html), which can help you interpret the results.
+
+{{site.data.alerts.callout_success}}
+Diagnostics will be collected a maximum of *N* times for a given activated fingerprint where *N* is the number of nodes in your cluster.
+{{site.data.alerts.end}}
+
+{% include common/sql/statement-bundle-warning.md %}
+
+#### Activate diagnostics collection and download bundles
+
+<img src="{{ 'images/v22.1/ui_activate_diagnostics.png' | relative_url }}" alt="Statements diagnostics" style="border:1px solid #eee;max-width:80%" />
+
+To activate diagnostics collection:
+
+1. Click the **Activate diagnostics** button. <span class="version-tag">New in v22.1</span> The **Activate statement diagnostics** dialog displays.
+
+    <img src="{{ 'images/v22.1/ui_activate_diagnostics_dialog.png' | relative_url }}" alt="Statements diagnostics" style="border:1px solid #eee;max-width:80%" />
+
+1. Choose whether to activate collection on the next statement execution (default) or if execution latency exceeds a certain time. If you choose the latter, accept the default latency of 100 milliseconds, or specify a different time.
```

Comment:
Can we call out the performance overhead for enabling diagnostics? All executions of the statement fingerprint will run slower with diagnostics enabled until diagnostics are collected.